fastlane/Fastfile

```diff
@@ -197,4 +197,49 @@ platform :ios do
     )
   end
 
+  desc "Build Perf-test app without Sentry"
+  lane :build_perf_test_app_plain do
+
+    setup_ci
+
+    sync_code_signing(
+      type: "appstore",
```

Comment:
I think this app should be OK to export as a development build instead of app store, unless there was a specific reason you went with that? If you copied this from `build_ios_swift`, that's the only test app we try exporting for the app store. All the others are done as development.
```suggestion
      type: "development",
```